### PR TITLE
Don't need ProxyCommand in SSH aliases

### LINKED ISSUE #933: Tails SSH doesn't need proxy command
On Tails you do not need to use torify or the proxy command for ssh connections to go over tor. That line can be taken out of the install directions for setting up the SSH host aliases for the admin workstation.

### PR DESCRIPTION
**Pull request by ageis**
Fixes #933

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -384,14 +384,12 @@ This step is optional but makes it much easier to connect to and administer the
...
   User <ssh_user>
-  ProxyCommand connect -R remote -5 -S localhost:9050 %h %p
 Host mon
   Hostname yt4j52ajfvbmvtc7.onion
   User <ssh_user>
-  ProxyCommand connect -R remote -5 -S localhost:9050 %h %p
 ```
 
-Now you can simply use `ssh app` and `ssh mon` to connect to each server, and it will be stored in the Tails Dotfiles persistence. You might see the message "Enter SOCKS5 password for amnesia@localhost" — this can be ignored, just always press Enter.
+Now you can simply use `ssh app` and `ssh mon` to connect to each server, and it will be stored in the Tails Dotfiles persistence.
```
Comment (garrettr) [inline docs/install.md]: But you don't recommend using `torify`, you're recommending just running `ssh {app,mon}`. Won't that still lead to the SOCKS5 prompt?

Comment (dolanjs) [inline docs/install.md]: Using torify on Tails spews out a bunch of errors (below) when using torify even though it still torify still works. So not sure what the best course of action is.

```
12:22:01 libtorsocks(6492): WARNING: The symbol res_query() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_query() with the reported error: Not Found
12:22:01 libtorsocks(6492): WARNING: The symbol res_search() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_search() with the reported error: Not Found
12:22:01 libtorsocks(6492): WARNING: The symbol res_send() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_send() with the reported error: Not Found
12:22:01 libtorsocks(6492): WARNING: The symbol res_querydomain() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_querydomain() with the reported error: Not Found
12:22:01 libtorsocks(6493): WARNING: The symbol res_query() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_query() with the reported error: Not Found
12:22:01 libtorsocks(6493): WARNING: The symbol res_search() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_search() with the reported error: Not Found
12:22:01 libtorsocks(6493): WARNING: The symbol res_send() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_send() with the reported error: Not Found
12:22:01 libtorsocks(6493): WARNING: The symbol res_querydomain() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_querydomain() with the reported error: Not Found
```

Comment (ageis) [inline docs/install.md]: @garrettr nope, while it still uses SOCKS, there is no prompt for regular SSH on Tails. The THS resolution is totally transparent.

Comment (ageis) [inline docs/install.md]: @garrettr scratch that, I was wrong .I will put that note back in.

Comment (ageis) [inline docs/install.md]: Actually not sure. @dolanjs says that everybody gets the prompt, and he definitely does, which is why I retracted my statement. But I'm not getting it. We should maybe do some checking to see whether this is consistent with the latest version of Tails.
amnesia_ssh

Comment (garrettr) [inline docs/install.md]: Can you test with the `~/.ssh/config` as set up by this PR (e.g. `ssh app`, not `kevin@xyz.onion`)? I've got a feeling the custom config is part of the reason why we're seeing the SOCKS prompt.

Comment (ageis) [inline docs/install.md]: @garrettr yeah, I still don't get it

Comment (ageis) [inline docs/install.md]: @garrettr the final verdict from me is that there's normally no SOCKS5 password prompt. I determined as such on Tails 1.3.1 with and without persistence, with and without an SSH identity, with and without an ~/.ssh/config file. You can merge.

Comment (garrettr) [inline docs/install.md]: @ageis Thanks for the due diligence! :+1:

**Pull request merged**